script/constants/deploymentConfig.ts

```diff
@@ -0,0 +1,62 @@
+import {
+  ChainSlug,
+  DeploymentAddresses,
+  DeploymentMode,
+  getAllAddresses,
+  MainnetIds,
+  S3ChainConfig,
+  S3Config,
+  TestnetIds,
+} from "@socket.tech/dl-core";
+import { getMode } from "./config";
+import { chainSlugReverseMap } from "../setup/enumMaps";
+
+export const DEPLOYMENT_CONFIG_URL =
+  "https://la9apjt8o9.execute-api.us-east-1.amazonaws.com/surge/v1/getS3Config";
```

Comment:
will update this with custom domain. cant put this in env, as outside teams will use it, will have to share this as well with them to add to .env. Also, provides public info.